source/linux/Foundational_Components/System_Security/SELinux.rst

```diff
@@ -0,0 +1,139 @@
+#######
+SELinux
+#######
+
+Introduction
+------------
+
+Security-Enhanced Linux (`SELinux <https://github.com/SELinuxProject>`_) is a project to implement mandatory access control under Linux. It is a set of kernel modifications and user-space tools that provide a mechanism for supporting access control security policies. This project was initially developed by the National Security Agency (NSA), as a reference implementation.
+
+SELinux can run in one of three modes: *enforcing*, *permissive*, or *disabled*.
+
+- Enforcing mode is the recommended mode of operation where SELinux operates normally, enforcing the loaded security policy on the entire system.
+- In permissive mode, the system acts as if SELinux is enforcing the loaded security policy, including labeling objects and emitting access denial entries in the logs, but it does not actually deny any operations. While not recommended for production systems, permissive mode can be helpful for SELinux policy development and debugging.
+- Disabled mode is strongly discouraged; not only does the system avoid enforcing the SELinux policy, it also avoids labeling any persistent objects such as files, making it difficult to enable SELinux in the future.
+
+.. note::
+
+   This guide assumes that the user understands how to build the Linux Kernel and, Yocto or Debian distribution.
+
+Filesystem Setup
+----------------
+
+Yocto
+^^^^^
+
+The following steps describe how to build SELinux user-space tools and configuration on Yocto. Please use :ref:`Processor SDK - Building the SDK with Yocto <building-the-sdk-with-yocto>` as reference.
+
+1. On a fresh build, update the config file to use :file:`configs/arago-scarthgap-selinux-config.txt`. This config includes the opensource `meta-selinux` layer into the build and also adds additional features for the filesystem.
+
+   .. note::
+
+      The default policy is targeted. If you want to update it, add the variable ``PREFERRED_PROVIDER_virtual/refpolicy = "refpolicy-<your_choice>"`` to the :file:`local.conf` where ``<your_choice>`` is targeted, mls, mcs.
+
+2. Build your custom image in build directory
```

Comment:
```suggestion
2. Build your :file:`tisdk-default-image` in build directory
```

This can't be called out as `custom image` since selinux features are only deployed in `tisdk-default-image` via dynamic layers - https://git.ti.com/cgit/arago-project/meta-arago/tree/meta-arago-distro/dynamic-layers/selinux/recipes-core/images?h=scarthgap